functions/snippets/src/main/java/FirebaseAuth.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START functions_firebase_auth]
+import com.google.cloud.functions.Context;
+import com.google.cloud.functions.RawBackgroundFunction;
+import java.util.logging.Logger;
+
+public class FirebaseAuth implements RawBackgroundFunction {
+
+  private static final Logger logger = Logger.getLogger(FirebaseAuth.class.getName());
+
+  @Override
+  public void accept(String json, Context context) {
```

Comment:
nit: Perhaps you should show the user how to parse the JSON?

Reply:
That would change the functionality of the snippet and would make Java different than the other languages.
I've added a stub for parsing JSON though.